code42.json

```diff
@@ -75,26 +93,28 @@
                     "allow_list": true
                 },
                 "start_time": {
```

Comment:
Note that the `on poll` parameters _have_ to be these, whereas the app configuration settings can have anything. Thus, apps put the polling-related ingest parameters there instead.

Another really weird thing is that there is no place really in the UI to specify the `start_time` and the `end_time`. It is assumed these will come from another source. In our case, it comes from the app configuration as well as the stored last timestamp ongoingly

Reply:
If it's only the `on poll` parameters that have their names fixed to `start_time` / `end_time`, can we change the names of `start_date` and `end_date` to more explicitly call out that they are only used for the initial poll? I keep getting confused about which of these two sets of variables is configured at the asset level vs. action level, and which overrides which.